config/main.py

```diff
@@ -2468,5 +2468,41 @@ def feature_status(name, state):
 
     config_db.mod_entry('FEATURE', name, {'status': state})
 
+#
+# 'container' group ('config container ...')
+#
+@config.group(name='container', invoke_with_command=False)
+def container():
+    """Config container"""
```

Comment:
Change help string to `Modify configuration of containers`

Also update this in Command reference PR.

Reply:
Fixed.